src/components/routing/routers/GameRouter.js

```diff
@@ -5,57 +5,63 @@ import BlackCardSelection from "../../views/BlackCardSelection";
 import PlayWhites from "components/views/PlayWhites";
 import Matches from "components/views/Matches";
 import RateWhites from "../../views/RateWhites";
+import UserHand from "../../views/UserHand";
 import Chat from "../../views/Chat";
 
 const GameRouter = props => {
-  /**
-   * "this.props.base" is "/app" because as been passed as a prop in the parent of GameRouter, i.e., App.js
-   */
-  return (
-    <div style={{display: 'flex', flexDirection: 'column'}}>
-
-      {/* menu */}
-      <Route exact path={`${props.base}/menu`}>
-        <GameMenu/>
-      </Route>
-
-      {/* black card selection */}
-      <Route exact path={`${props.base}/select/blackCard`}>
-          <BlackCardSelection />
-      </Route>
-
-      {/* play white cards */}
-      <Route exact path={`${props.base}/playWhites`}>
-        <PlayWhites/>
-      </Route>
-
-      {/* matches */}
-      <Route exact path={`${props.base}/matches`}>
-        <Matches/>
-      </Route>
-
-      {/* rate white cards */}
-      <Route exact path={`${props.base}/rateWhites`}>
-        <RateWhites/>
-      </Route>
-
-        <Route exact path={`${props.base}/chat`}>
-            <Chat/>
-        </Route>
-
-      {/* fallback option */}
-      <Route exact path={`${props.base}`}>
-        <Redirect to={`${props.base}/menu`}/>
-      </Route>
-    </div>
-  );
+    /**
+    * "this.props.base" is "/app" because as been passed as a prop in the parent of GameRouter, i.e., App.js
+    */
+    return (
+        <div style={{display: 'flex', flexDirection: 'column'}}>
+
+            {/* menu */}
+            <Route exact path={`${props.base}/menu`}>
+                <GameMenu/>
+            </Route>
+
+            {/* black card selection */}
+            <Route exact path={`${props.base}/select/blackCard`}>
+                <BlackCardSelection />
+            </Route>
+
+            {/* play white cards */}
+            <Route exact path={`${props.base}/playWhites`}>
+                <PlayWhites/>
+            </Route>
+
+            {/* matches */}
+            <Route exact path={`${props.base}/matches`}>
+                <Matches/>
+            </Route>
+
+            {/* hand */}
+            <Route exact path={`${props.base}/hand`}>
+                <UserHand/>
+            </Route>
+
+            {/* rate white cards */}
+            <Route exact path={`${props.base}/rateWhites`}>
+                <RateWhites/>
+            </Route>
+
+            <Route exact path={`${props.base}/chat`}>
+                <Chat/>
+            </Route>
+
+            {/* fallback option */}
+            <Route exact path={`${props.base}`}>
+                <Redirect to={`${props.base}/menu`}/>
+            </Route>
+        </div>
+    );
```

Comment:
For future: if you don't change the code itself, then don't format it in a issue_branch, because it makes it harder to filter out the code that you actually wrote.. (Also, most of us have set 2 spaces for the frontend, so you might want to change that too)
![image](https://user-images.githubusercontent.com/75019468/168476831-69f1ec65-439d-45d4-8686-8bc864a102db.png)

Reply:
moved it back to 2 spaces..